sdk/search/Directory.Build.props

```diff
@@ -3,7 +3,8 @@
   <Choose>
     <When Condition="'$(IsDataPlaneProject)' == 'true'">
       <PropertyGroup>
-        <VersionPrefix>10.0.0</VersionPrefix>
+        <VersionPrefix>10.0.1</VersionPrefix>
```

Comment:
As an FYI we are starting to versioning guidance we have outline at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#package-versioning. Which generally means we should bump the minor version after a QA release so that we always have versioning space for hotfixes it they are necessary. Assuming there aren't any other changes to search then I think this would qualify as a hotfix but I wanted to at least point out our docs if you guys haven't had the chance to review them yet.

Reply:
Thanks for the pointer. I think this qualifies as a hotfix, although we didn't follow the branching guidance this time.

Reply:
We also have document on the branching workflows at https://github.com/Azure/azure-sdk/blob/master/docs/policies/repobranching.md#hotfix-branches incase you guys haven't seen that either.